src/Encryptor/LegacyEncryptor.php

```diff
@@ -5,32 +5,24 @@
 use Carnage\EncryptedColumn\Exception\PopArtPenguinException;
 use Carnage\EncryptedColumn\ValueObject\EncryptorIdentity;
 use Carnage\EncryptedColumn\ValueObject\IdentityInterface;
+use Carnage\EncryptedColumn\ValueObject\Key;
 use phpseclib\Crypt\Base;
 use phpseclib\Crypt\Rijndael;
 
 class LegacyEncryptor implements EncryptorInterface
 {
     const IDENTITY = 'legacy';
-    /**
-     * @var string
-     */
-    private $secret;
 
-    public function __construct($secret)
-    {
-        $this->secret = $secret;
-    }
```

Comment:
Might it not be better to have a default set through the constructor and make the `Key` arguments below optional?